DEPRECATIONS.md

```diff
@@ -1,7 +1,9 @@
 # Deprecation Notices
 
-*Deprecated in 6.2.0*
+_Deprecated in 6.2.0_
+
 - `modular-spacing` mixin will be removed in Version 7. Please use `responsive-spacing` instead.
 - Noto Sans Font will be removed in Version 7
 - Kalam Font will be removed in Version 7.
 - `su-brand-bar` will be removed in Version 7, and replaced with `su-global-identity-bar`.
+- Stanford University lockup options C, G, J, and Q (i.e. `.su-lockup--option-c`) will be removed in Version 7.
```

Comment:
This notice should be moved to a new heading `Deprecated in 6.2.2` as 6.2.1 has already been released.

Also, I do not see a deprecation comment in the SCSS eg: https://github.com/SU-SWS/decanter/blob/master/core/src/scss/components/lockup/_lockup--option-c.scss

Please add deprecation notice SCSS comment to each of the lockups. EG: https://github.com/SU-SWS/decanter/blob/master/core/src/scss/core/fonts/_fonts.scss#L21 and https://github.com/SU-SWS/decanter/blob/master/core/src/scss/utilities/mixins/modular-scale/_modular-spacing.scss#L22